### DISCUSSION TITLE
How to show the repository config settings?

### DISCUSSION THREAD

**Discussion by KyrietS** [rustic-rs/rustic#1331] [Q&A] [answered]
Hi, I created a new repo:
```
rustic init -r repo --password 123
```

And then I set some config values. For example:

```
rustic -r repo --password 123 config --set-version 2
rustic -r repo --password 123 config --set-compression 10
rustic -r repo --password 123 config --set-min-packsize-tolerate-percent 25
```

My question is: **How can I check what config values are currently set?** (repo version, compression level, etc.)

Despite it's name, the `rustic show-config` does not show the config I've just set at all:

```
rustic -r repo --password 123 show-config
...
[repository]
repository = "repo"
password = "123"
no-cache = false
warm-up = false

[repository.options]
...
```

Thanks!

**Comment by kapitainsky** [comment, marked as answer]
```
rustic cat config
```

**Reply by aawsome** [reply]
I'd like to add that this shows the content of the config file as saved in the repository. Note that this does not show which defaults are used if there are some options not present in the config file. To see the defaults, use `rustic config -h`.

**Reply by KyrietS** [reply]
Thank you! That answers my question 🤗
